packages/ts/src/components/graph/index.ts

```diff
@@ -116,7 +120,11 @@ export class Graph<
   }
 
   public get selectedNode (): GraphNode<N, L> {
```

Comment:
@reb-dev I agree with your comment regarding marking the `selectedNode` things as deprecated. You can use this jsdoc tag: https://jsdoc.app/tags-deprecated